climada/hazard/base.py

```diff
@@ -513,12 +513,14 @@ def from_xarray_raster(
 
             Default values are:
 
-            * ``date``: The ``event`` coordinate interpreted as date
+            * ``date``: The ``event`` coordinate interpreted as date or ordinal, or
+              zeros if that fails (which will issue a warning).
```

Comment:
I am not sure about the value `0` since this is then not a valid ordinal. Thus, one would get a non-valid `event_date` which then creates easily problems down the line when a method expects an ordinal. This then might lead to hard-to-debug/understand error messages for users. Any other idea?

Reply:
You are correct. I actually messed up the test. It is fixed now. I also did not realize that 0 does not work as ordinal. I now chose 1 as default value, the ordinal of date 01.01.0001.